pandas/tests/frame/methods/test_sort_index.py

```diff
@@ -35,15 +35,15 @@ def test_sort_index_and_reconstruction_doc_example(self):
             ),
         )
         result = df.sort_index()
-        assert result.index.is_lexsorted()
+        assert result.index._is_lexsorted()
```

Comment:
for the cases where these are the same as is_monotonic (vast majority), ok with removing the _is_lexsorted assert